core/input/input.cpp

```diff
@@ -1356,8 +1356,9 @@ void Input::parse_mapping(String p_mapping) {
 
 		String output = entry[idx].get_slice(":", 0).replace(" ", "");
 		String input = entry[idx].get_slice(":", 1).replace(" ", "");
-		ERR_CONTINUE_MSG(output.length() < 1 || input.length() < 2,
-				vformat("Invalid device mapping entry \"%s\" in mapping:\n%s", entry[idx], p_mapping));
+		if (output.length() < 1 || input.length < 2) {
+      continue;
```

Comment:
You need to follow the proper indentation, easiest to use `clang-format`, or set up your IDE with the correct indentation, [see here](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html)